source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,192 @@
+.. _realm-sync-migrate-modes:
+
+=============================================
+Migrate Partition-Based Sync to Flexible Sync
+=============================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+You can automatically migrate App Services Apps that use Partition-Based Sync
+to Flexible Sync. We recommend doing this, as Partition-Based Sync is an older
+mode.
+
+Other than upgrading the SDK version, migrating sync modes doesn't require any 
+changes to your client app code. Your App's backend will automatically translate
+the client's Partition-Based Sync requests into Flexible Sync.
+.. QUESTION: is there a performance hit for the translation?
```

Comment:
nope! there is a one-time cost (roughly equivalent to the regular cost of a client reset) when the PBS client first tries to connect, but once that handshake happens it will be a regular flexible sync client moving forward